lib/oli/groups/community.ex

```diff
@@ -3,19 +3,20 @@ defmodule Oli.Groups.Community do
   import Ecto.Changeset
 
   schema "communities" do
-    field :name, :string
-    field :description, :string
-    field :key_contact, :string
-    field :global_access, :boolean, default: true
+    field(:name, :string)
```

Comment:
I don't like the `()` in the field functions 🙁

Reply:
@mcass19 @jrissler I don't like them either, my formatter adds them on save and it's an annoying thing I didn't figure out yet. I think you @mcass19 were experiencing the same, were you able to adjust it to not add them?